pkg/cluster/deployresources_resources.go

```diff
@@ -94,7 +94,7 @@ func dnsVirtualNetworkLink(vnetID string, installConfig *installconfig.InstallCo
 	}
 }
 
-func networkPrivateLinkService(infraID, subscriptionID string, oc *api.OpenShiftCluster, installConfig *installconfig.InstallConfig) *arm.Resource {
+func (m *manager) networkPrivateLinkService(infraID, subscriptionID string, oc *api.OpenShiftCluster, installConfig *installconfig.InstallConfig) *arm.Resource {
```

Comment:
as part of these changes the oc argument can be removed - please do so.  Check in case there are other arguments that can be removed?

Reply:
Done